File: etl/models/candidato.py
from sqlalchemy import Column, String, Integer, Date
from .base import Base

class Candidato(Base):
    __tablename__ = "candidato"

    nr_cpf_candidato = Column(String(11), primary_key=True)
    nm_candidato = Column(String, nullable=False)
    nm_social_candidato = Column(String)
    nm_urna_candidato = Column(String)
    dt_nascimento = Column(Date)
    sg_uf_nascimento = Column(String(2))
    cd_genero = Column(Integer)
    ds_genero = Column(String)
    cd_grau_instrucao = Column(Integer)
    ds_grau_instrucao = Column(String)
    cd_estado_civil = Column(Integer)
    ds_estado_civil = Column(String)
    cd_cor_raca = Column(Integer)
    ds_cor_raca = Column(String)

File: src/backend/VoteBem/Entities/Candidato.cs
namespace VoteBem.Entities
{
    public class Candidato
    {
        public string NrCpfCandidato { get; private set; } = null!;
        public string NmCandidato { get; private set; } = null!;
        public string? NmSocialCandidato { get; private set; }
        public string? NmUrnaCandidato { get; private set; }
        public DateOnly? DtNascimento { get; private set; }
        public string? SgUfNascimento { get; private set; }
        public int? CdGenero { get; private set; }
        public string? DsGenero { get; private set; }
        public int? CdGrauInstrucao { get; private set; }
        public string? DsGrauInstrucao { get; private set; }
        public int? CdEstadoCivil { get; private set; }
        public string? DsEstadoCivil { get; private set; }
        public int? CdCorRaca { get; private set; }
        public string? DsCorRaca { get; private set; }

        public ICollection<Candidatura> Candidaturas { get; private set; } = [];

        protected Candidato() { }
    }
}
